src/main/java/me/crespel/runtastic/model/SportSession.java

```diff
@@ -60,9 +62,66 @@
 	@JsonIgnore
 	private List<GpsData> gpsData;
 
+	@JsonIgnore
+	private GpxType gpx;
+
 	@JsonIgnore
 	private List<HeartRateData> heartRateData;
 
+	@JsonIgnore
+	private SportSessionAlbums SessionAlbum;

```

Comment:
Variable name should start in lowercase:
```suggestion
	private SportSessionAlbums sessionAlbum;
```

Reply:
Thanks, fixed!